src/components/LoginComponent.jsx

```diff
@@ -1,64 +1,74 @@
-import React from 'react';
-import { withRouter } from 'react-router-dom';
-import { Button, Container, Header, Image } from 'semantic-ui-react';
-import { SemanticToastContainer } from 'react-semantic-toasts';
+import React from "react";
```

Comment:
Single quotes please. This implies you haven't yet rebased with develop. The rule has been updated to single quotes for strings and double quotes for jsx